ruby_on_rails/template.rb

```diff
@@ -0,0 +1,33 @@
+# replace rubocop-rails-omakase with renuocop
+
+gsub_file "Gemfile", /gem "rubocop-rails-omakase"/, "gem \"renuocop\""
+
+# override the content of .rubocop.yml
+create_file ".rubocop.yml", force: true do
+  <<~RUBOCOP
+    inherit_gem:
+      renuocop: config/base.yml
+  RUBOCOP
+end
+
+after_bundle do
+  run "bundle exec rubocop -A"
```

Comment:
I changed it to run the "risky" version with `-A`. We can take this risk in a project setup